unity-client/Assets/Scripts/MainScripts/DCL/Controllers/InputController/InputController.cs

```diff
@@ -256,6 +290,13 @@ public static bool PassModifiers(Modifier modifiers)
         return true;
     }
 
+    /// <summary>
+    /// Process an input action mapped to a keyboard key.
+    /// </summary>
+    /// <param name="action">Trigger Action to perform</param>
+    /// <param name="key">KeyCode mapped to this action</param>
+    /// <param name="modifierKeys">KeyCodes required to perform the action</param>
+    /// <param name="modifiers">Miscellaneous modifiers required for this action</param>
```

Comment:
What are miscellaneous modifiers? Can we provide an example?

Reply:
They can be anything: having renderer enabled, mouse locked, not jumping or whatever you can imagine. I will provide a more clear description